awx/main/scheduler/task_manager.py

```diff
@@ -489,11 +494,18 @@ def generate_dependencies(self, undeped_tasks):
 
     def process_pending_tasks(self, pending_tasks):
         running_workflow_templates = set([wf.unified_job_template_id for wf in self.get_running_workflow_jobs()])
+        tasks_to_update_job_explanation = []
         for task in pending_tasks:
             if self.start_task_limit <= 0:
                 break
-            if self.is_job_blocked(task):
-                logger.debug("{} is blocked from running".format(task.log_format))
+            blocked_by = self.job_blocked_by(task)
+            if blocked_by:
+                task.log_lifecycle("blocked", blocked_by=blocked_by)
+                job_explanation = gettext_noop(f"waiting for {blocked_by._meta.model_name}-{blocked_by.id} to finish")
```

Comment:
Is it possible to make the blocked by job name to become a link in the UI so we can easily navigate to it and follow if we want to?

Reply:
exactly, we plan to make this a link in the UI, which would be helpful to users